diff --git a/src/config.rs b/src/config.rs
index 707bbe0..1c1cac1 100644
--- a/src/config.rs
+++ b/src/config.rs
@@ -117,8 +117,8 @@ const CHARS: &[char] = &[
     'm', 'n', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z',
 ];
 
-pub const RENDEZVOUS_SERVERS: &[&str] = &["rs-ny.rustdesk.com"];
-pub const RS_PUB_KEY: &str = "OeVuKk5nlHiXp+APNn0Y3pC1Iwpwn44JGqrQCsWqmBw=";
+pub const RENDEZVOUS_SERVERS: &[&str] = &["__RD_SERVER__"];
+pub const RS_PUB_KEY: &str = "__RD_PUBKEY__";
 
 pub const RENDEZVOUS_PORT: i32 = 21116;
 pub const RELAY_PORT: i32 = 21117;
 
-            custom_rendezvous_server: false,
+            custom_rendezvous_server: true,
